server/graphql/common/user.ts

```diff
@@ -36,9 +37,9 @@ export const createUser = (
     location: Record<string, unknown>;
   },
   { organizationData, sendSignInLink, throwIfExists, redirect, websiteUrl, creationRequest }: CreateUserOptions,
-): Promise<{ user: typeof models.User; organization?: typeof models.Collective }> => {
+): Promise<{ user: User; organization?: typeof models.Collective }> => {
   return sequelize.transaction(async transaction => {
-    let user = await models.User.findOne({ where: { email: userData.email.toLowerCase() } }, { transaction });
```

Comment:
:heart: typescript

Reply:
These small errors will be easily prevented!